resources/views/layout.blade.php

```diff
@@ -55,75 +55,112 @@
+                            <li class="col-sm-12">
                                 <ul>
+                                    <li class="dropdown-header">Membership</li>
+                                    <li>{!! link_to_route("visiting.landing", "Visit/Transfer") !!}</li>
+                                    <li>{!! link_to_route("mship.email", "Email a Member") !!}</li>
+                                    <li class="divider"></li>
                                     <li class="dropdown-header">Third-Party Services</li>
                                     <li>{!! link_to_route("teamspeak.new", "TS Registration") !!}</li>
                                     <li>{!! link_to_route("slack.new", "Slack Registration") !!}</li>
-                                    <li class="divider"></li>
-                                    <li class="dropdown-header">Community Features</li>
-                                    <li>{!! link_to_route("mship.notification.list", "View Notifications") !!}</li>
-                                    <li>{!! link_to_route("mship.email", "Email a Member") !!}</li>
-                                    <li>{!! link_to_route("mship.feedback.new", "Controller/Pilot Feedback") !!}</li>
                                 </ul>
                             </li>
                         </ul>
-
                     </li>
                 </ul>
 
-                @if(Auth::check())
-                    <ul class="nav navbar-nav navcustom">
+                @if(Auth::check() || Session::has('auth.vatsim-sso'))
+                    {!! Form::open(['route' => 'logout', 'id' => 'logout-form']) !!}
+                    <ul class="nav navbar-nav navcustom navbar-right account-dropdown">
                         <li class="dropdown dropdown-large">
-                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">Account <b class="caret"></b></a>
-                            <ul class="dropdown-menu dropdown-menu-large row mainmenu_dropdown">
-                                <li class="col-sm-6">
+                            <a href="#" class="dropdown-toggle hidden-xs hidden-sm" data-toggle="dropdown">{{Auth::user()->name.' (' .Auth::user()->id.')'}} <b class="caret"></b></a>
+							<a href="#" class="dropdown-toggle visible-xs visible-sm" data-toggle="dropdown"><i class="fa fa-sliders"></i> <b class="caret"></b></a>
+                            <ul class="dropdown-menu dropdown-menu-logout dropdown-menu-large row mainmenu_dropdown">
+                                <li class="col-sm-12">
                                     <ul>
-                                        <li class="dropdown-header">Password</li>
-                                        <li>{!! link_to_route('password.change', "Modify") !!}</li>
+                                        <li><a>{{ Auth::user()->status_string }} {{ !is_null(Auth::user()->primary_state) ? Auth::user()->primary_state->name : 'unknown state' }} Member</a></li>
```

Comment:
I think the division state and ratings can be removed from the account dropdown - what do you think @atoff?